mimer-config/src/main/java/com/avanza/astrix/config/PropertyParser.java

```diff
@@ -15,15 +15,25 @@
  */
 package com.avanza.astrix.config;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 interface PropertyParser<T> {
-	
+
 	public static PropertyParser<Boolean> BOOLEAN_PARSER = new BooleanParser();
 	public static PropertyParser<String> STRING_PARSER = new StringParser();
 	public static PropertyParser<Long> LONG_PARSER = new LongParser();
 	public static PropertyParser<Integer> INT_PARSER = new IntParser();
+	public static PropertyParser<List<String>> STRING_LIST_PARSER = new ListParser<>(Function.identity());
+	public static PropertyParser<List<Integer>> INT_LIST_PARSER = new ListParser<>(Integer::parseInt);
+	public static PropertyParser<List<Long>> LONG_LIST_PARSER = new ListParser<>(Long::parseLong);
+	public static PropertyParser<List<Boolean>> BOOLEAN_LIST_PARSER = new ListParser<>(Boolean::parseBoolean);
```

Comment:
Sorry for being picky, but I think these should all use the respective `valueOf`. The return value of `valueOf` is the object type, while the `parse*` methods all return primitives which will then be immediately boxed.
```suggestion
	public static PropertyParser<List<Integer>> INT_LIST_PARSER = new ListParser<>(Integer::valueOf);
	public static PropertyParser<List<Long>> LONG_LIST_PARSER = new ListParser<>(Long:: valueOf);
	public static PropertyParser<List<Boolean>> BOOLEAN_LIST_PARSER = new ListParser<>(Boolean:: valueOf);
```

Reply:
Also note that the single value `DynamicBooleanProperty` is not case sensitive when parsing, which the list variant is. That is probably fine though.